constants/currency.ts

```diff
@@ -75,7 +66,7 @@ export const MARKET_HOURS_SYNTHS = new Set([
 	...AFTER_HOURS_SYNTHS,
 ]);
 
-export const COMMODITY_SYNTHS = new Set<CurrencyKey>([Synths.sXAU, Synths.sXAG, Synths.sOIL]);
```

Comment:
There should be nothing in the dapp that hardcodes anything like this. If you need it, ask the synthetix protocol and it can be added to the core functionality exported by the `synthetix` module.

Reply:
Yeah good call. In the future if we add synths that require special functionality (market hours) we should define it somewhere in `synthetix`